docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,12 @@ develop
     *    - Type
          - Change
 
+
+    *    - |improved|
+         - ``TransactionManager`` now has an API ``getKeyValueServiceStatus`` that can return the health of the underlying KVS. This is designed for applications
```

Comment:
Should add documentation. What does each status mean, and what is the application developer expected to do to handle each case?